File: cpp/Estruturas repetitivas/sequencia_impares.cpp
#include <iostream>
using namespace std;

int main() {

    int X;

    cout << "Digite um valor: ";
    cin >> X;

    for (int i = 1; i <= X; i++) {

        if (i % 2 != 0) {
            cout << i << endl;
        }
    }

    return 0;
}

File: java/Estruturas repetitivas/SequenciaImpares.java
import java.util.Scanner;

public class SequenciaImpares {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int X;

        System.out.println("Digite um valor:");
        X = sc.nextInt();

        for (int i = 1; i <= X; i++) {

            if (i % 2 != 0) {
                System.out.println(i);
            }

        }

        sc.close();
    }
}